src/components/form/edit/section.vue

```diff
@@ -81,24 +81,27 @@ $heading-margin-bottom: 10px;
   > :nth-child(2) {
     flex-grow: 1;
     overflow-x: hidden;
-    padding-top: 16px;
+    padding-top: 6px;
   }
 }
 
 .form-edit-section-icon-container {
+  @include icon-box;
+  font-size: 24px;
+  padding-inline: 0;
```

Comment:
This zeroes out the horizontal padding from `icon-box`, but not the vertical padding. We want the box to be a fixed width so that the different section icons all line up. But the width of different icons vary quite a bit. In some cases, I think it works best for the icon to eat into the padding. To do so, we remove the horizontal padding and let the fixed with do all the work.